source/agora/test/Base.d

```diff
@@ -1832,19 +1923,8 @@ private immutable(Block) makeGenesisBlock (in KeyPair[] key_pairs,
 *******************************************************************************/
 
 private immutable(Block)[] generateBlocks (
-    ref immutable Block gen_block, size_t count)
+    ref immutable Block gen_block)
 {
```

Comment:
We don't really need this function anymore. How about inlining the code?

Reply:
It is first commit of many. It will go.

Reply:
Gone:-)